src/collective/importexport/browser/import_view.py

```diff
@@ -492,6 +553,27 @@ def updateWidgets(self):
         self.fields['header_mapping'].field.bind(self.context)
         super(ImportForm, self).updateWidgets()
 
+    def getObjectsToExport(self):
+        # Extract form field values and errors from HTTP request
+        data, errors = self.extractData()
+        if errors:
+            return False
+        container = self.context
+        container_path = "/".join(container.getPhysicalPath())
+        #TODO: should we allow more criteria? or at least filter by type?
+        query = dict(path={"query": container_path, "depth": 1},
+#                    portal_type=object_type,
+                     )
+#        query[primary_key]=key_arg[primary_key]
+
+        # blank header or field means we don't want it
```

Comment:
Basically, when you select columns, you also select column heading title. If you leave it empty it is considered no needed. This was done before.